go/oasis-test-runner/env/env.go

```diff
@@ -52,6 +53,27 @@ func (pfs *ParameterFlagSet) MarshalJSON() ([]byte, error) {
 	return json.Marshal(ps)
 }
 
+// Clone clones the parameter flagset.
+//
+// This function is usually called when cloning scenarios where we also want to clone existing
+// parameter values.
+func (pfs *ParameterFlagSet) Clone() *ParameterFlagSet {
+	// XXX: pfs.name and pfs.errorHandling are private. Use reflection.
```

Comment:
Is this just to get the two parameters from the constructor? Why not add a `NewParameterFlagSet` function and record these two arguments in `ParameterFlagSet` instead?